chapters/ch16.md

```diff
@@ -1,70 +1,71 @@
 # 16. Cleaning up mislabeled dev and test set examples
 ->
-
+# 16. Dọn dẹp những mẫu bị gán nhãn nhầm trong tập phát triển và tập kiểm tra
 
 During error analysis, you might notice that some examples in your dev set are mislabeled. When I say “mislabeled” here, I mean that the pictures were already mislabeled by a human labeler even before the algorithm encountered it. I.e., the class label in an example ​(x,y)​ has an incorrect value for ​y.​ For example, perhaps some pictures that are not cats are mislabeled as containing a cat, and vice versa. If you suspect the fraction of mislabeled images is significant, add a category to keep track of the fraction of examples mislabeled:
 ->
+Trong quá trình phân tích lỗi, bạn có thể nhận thấy rằng một vài mẫu trong trong tập phát triển đã bị gán nhãn nhầm. Khi tôi nói "bị gán nhãn nhầm" ở đây, ý tôi là những tấm ảnh đã bị gán nhãn nhầm bởi người dán nhãn trước cả khi được đưa vào thuật toán. Hay nói cách khác, nhãn lớp của một mẫu (x,y) có giá trị sai cho y.  Ví dụ, có thể một số ảnh không chứa mèo bị gán nhãn nhầm thành có mèo và ngược lại. Nếu bạn nghi ngờ rằng tỷ lệ những ảnh bị gán nhãn nhầm là đáng kể, hãy thêm một hạng mục để theo dõi tỷ lệ các mẫu bị gán nhãn nhầm:
 
 ![img](../imgs/C16_01.png)
 
 
 Should you correct the labels in your dev set? Remember that the goal of the dev set is to help you quickly evaluate algorithms so that you can tell if Algorithm A or B is better. If the fraction of the dev set that is mislabeled impedes your ability to make these judgments, then it is worth spending time to fix the mislabeled dev set labels.
 ->
-
+Vậy bạn có nên sửa lại những nhãn sai trong tập phát triển không? Hãy nhớ rằng mục tiêu của tập phát triển là giúp bạn nhanh chóng đánh giá những thuật toán nhờ đó bạn có thể biết liệu Thuật toán A hay Thuật toán B là tốt hơn. Nếu tỷ lệ bị gán nhãn nhầm trong tập phát triển cản trở khả năng ra những quyết định này của bạn, thì sẽ là đáng để bỏ thời gian ra để sửa lại những nhãn bị gán nhầm của tập phát triển.
 
 For example, suppose your classifier’s performance is:
 ->
-
+Để ví dụ, giả sử hiệu quả bộ phân loại của bạn là:
 
 * Overall accuracy on dev set.................... 90% (10% overall error.)
 ->
-
+* Độ chính xác chung trên tập phát triển.................... 90% (10% lỗi chung.)
```

Comment:
Từ "overall" dịch là "chung" dễ gây khó hiểu. Bản thân từ "chung" trong tiếng Việt cũng có thể hiểu theo nhiều hơn một nghĩa.

Mình nghĩa từ "tổng thể" hoặc "toàn bộ" tốt hơn trong trường hợp này, 
"overall error" -> "lỗi tổng thể"
"overall accuracy" -> "độ chính xác tổng thể"